Lesson-004/Lesson-004.R

```diff
@@ -5,7 +5,7 @@ library(dplyr)
 # Load the file Lesson-004/data/cervix.csv and store it in a variable called 
 # "cervix": 
 
-
+cervix<-read.csv("Lesson-004/data/cervix.csv")
```

Comment:
Right, I think I left that in by mistake, I just did it to look at the dataset

From: Sean Hughes [mailto:notifications@github.com]
Sent: Monday, May 16, 2016 11:53 AM
To: seaaan/R-without-tears
Cc: Claire Levy; Author
Subject: Re: [seaaan/R-without-tears] added some hw questions (#4)


In Lesson-004/Lesson-004.R<https://github.com/seaaan/R-without-tears/pull/4#discussion_r63404910>:

> @@ -5,7 +5,7 @@ library(dplyr)

>  # Load the file Lesson-004/data/cervix.csv and store it in a variable called

>  # "cervix":

>

> -

> +cervix<-read.csv("Lesson-004/data/cervix.csv")

I had been leaving the read.csv line blank to make them fill it in as a reminder of how to do it. Otherwise great!

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/seaaan/R-without-tears/pull/4/files/1c2f7727f05422e9631c4b6d974af5564a3f9f7a#r63404910>